modules/assisted-installer-using-the-assisted-installer.adoc

```diff
@@ -6,44 +6,6 @@
-
-* *Web user interface:* The web user interface performs cluster installation without the user having to create the installation configuration files manually.
-* *No bootstrap node:* A bootstrap node is not required when installing with the {ai-full}. The bootstrapping process executes on a node within the cluster.
-* *Hosting:* The {ai-full} hosts:
-** Ignition files
-** The installation configuration
-** A discovery ISO
-** The installer
-* *Streamlined installation workflow:* Deployment does not require in-depth knowledge of {product-title}. The {ai-full} provides reasonable defaults and provides the installer as a service, which:
-** Eliminates the need to install and run the {product-title} installer locally.
-** Ensures the latest version of the installer up to the latest tested z-stream releases. Older versions remain available, if needed.
-** Enables building automation by using the API without the need to run the {product-title} installer locally.
-* *Advanced networking:* The {ai-full} supports IPv4 networking with SDN and OVN, IPv6 and dual stack networking with OVN only, NMState-based static IP addressing, and an HTTP/S proxy. OVN is the default Container Network Interface (CNI) for OpenShift Container Platform 4.12 and later, but you can use SDN.
-
-* *Preinstallation validation:* The {ai-full} validates the configuration before installation to ensure a high probability of success. The validation process includes the following checks:
-** Ensuring network connectivity
-** Ensuring sufficient network bandwidth
-** Ensuring connectivity to the registry
-** Ensuring time synchronization between cluster nodes
-** Verifying that the cluster nodes meet the minimum hardware requirements
-** Validating the installation configuration parameters
-* *REST API:* The {ai-full} has a REST API, enabling automation.
-
-The {ai-full} supports installing {product-title} on premises in a connected environment, including with an optional HTTP/S proxy. It can install the following:
-
-* Highly available {product-title} or Single Node OpenShift (SNO)
-+
-[NOTE]
-====
-SNO is not supported on {ibmzProductName} and {ibmpowerProductName}.
-====
-+
-* {product-title} on bare metal, Nutanix, or vSphere with full platform integration, or other virtualization platforms without integration
-* Optionally {VirtProductName} and {rh-storage} (formerly OpenShift Container Storage)
-
-The user interface provides an intuitive interactive workflow where automation does not exist or is not required. Users may also automate installations using the REST API.
+The {ai-full} is a user-friendly installation solution offered on the link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[Red Hat Hybrid Cloud Console]. The {ai-full} reduces installation complexity, enabling users to deploy and configure {product-title} clusters with minimal manual intervention.
```

Comment:
```suggestion
The {ai-full} is a user-friendly installation solution offered on the link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[Red{nbsp}Hat Hybrid Cloud Console]. The {ai-full} reduces installation complexity, so that you can deploy and configure {product-title} clusters with minimal manual intervention.
```

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces

![Uploading Screenshot From 2025-04-10 10-08-28.png…]()